github3/pulls.py

```diff
@@ -254,6 +256,9 @@ def is_merged(self):
 
         :returns: bool
         """
+        if self.merged:
```

Comment:
if it is not None. If it is `False` or `True` it came back from the API. You may not want to do this at all though because the cached information can be out of date.